addons/gamification/models/gamification_karma_rank.py

```diff
@@ -17,11 +17,28 @@ class KarmaRank(models.Model):
     karma_min = fields.Integer(string='Required Karma', required=True, default=1,
         help='Minimum karma needed to reach this rank')
     user_ids = fields.One2many('res.users', 'rank_id', string='Users', help="Users having this rank")
+    rank_users_count = fields.Integer("Nb. Users", compute="_compute_rank_users_count")
 
     _sql_constraints = [
         ('karma_min_check', "CHECK( karma_min > 0 )", 'The required karma has to be above 0.')
     ]
 
+    @api.depends('user_ids')
+    def _compute_rank_users_count(self):
+        for record in self:
+            record.rank_users_count = len(record.user_ids)
```

Comment:
Why do you not use read_group?

Reply:
! @Esperax

Reply:
@Esperax This is also an important comment, don't ignore it, check with @JKE-be if you don't understand how you should do it.
Also: the runbot of this branch is red, please check asap to fix it.